localization/pose_estimator/ndt_scan_matcher/ndt_scan_matcher/src/ndt_scan_matcher_core.cpp

```diff
@@ -710,13 +728,14 @@ bool NDTScanMatcher::getTransform(
   }
 
   try {
-    *transform_stamped_ptr =
-      tf2_buffer_.lookupTransform(target_frame, source_frame, ros::Time(0), ros::Duration(1.0));
+    *transform_stamped_ptr = tf2_buffer_.lookupTransform(
+      target_frame, source_frame, rclcpp::Time(0), rclcpp::Duration(1.0));
```

Comment:
Could this be `tf2::TimePointZero` or do they just mean the same thing?

Reply:
Yes, I should make it `tf2::TimePointZero`. And actually this should be another `waitForTransform` because `lookupTransform` with a timeout doesn't work.

Reply:
Actually I was going to ask this I did some digging. When I first raised up this issue with the waitForTransform with timeouts I couldn't find it in any of the documentation for tf2::BufferCore and I got compilation errors. But it seems like it is available for `tf2_ros::Buffer` which is what we are using: http://docs.ros2.org/foxy/api/tf2_ros/classtf2__ros_1_1Buffer.html#a3ab502cc1e8b608957a96ad350815aee

Reply:
Which discussion? If it's not not the porting document, could you link me to it?

Reply:
Sorry I didn't see this reply, here is the link to the discussion: https://gitlab.com/autowarefoundation/autoware.auto/AutowareAuto/-/issues/644